docker/prod/Dockerfile-nginxexporter

```diff
@@ -0,0 +1,6 @@
+# This is a base nginx image with Wikia's directory structure
+FROM quay.io/martinhelmich/prometheus-nginxlog-exporter
+
+COPY ./prometheus-nginxlog-exporter.hcl /etc/prometheus-nginxlog-exporter.hcl
```

Comment:
We should be able to mount the configuration file directly using method like this:
https://github.com/martin-helmich/prometheus-nginxlog-exporter#docker
This will allow us to use the original image and limit the work needed to get this running.

```bash
$ docker run --name nginx-exporter -p 4040:4040 -v logs:/mnt/nginxlogs -v /path/to/config.hcl:/etc/prometheus-nginxlog-exporter.hcl quay.io/martinhelmich/prometheus-nginxlog-exporter -config-file /etc/prometheus-nginxlog-exporter.hcl
```